apps/app-article-server/src/server/index.js

```diff
@@ -1,27 +1,30 @@
-var express = require("express");
-var React = require("react");
-var ReactDOM = require("react-dom/server");
-var app = express();
+const express = require("express");
+const React = require("react");
+const ReactDOM = require("react-dom/server");
+const app = express();
 const port = 8080;
-require("dotenv").config();
-import generateHtml from "./generateHtml";
-import Article from "../browser/components/article";
-import ErrorPage from "../browser/components/error";
+const axios = require("axios");
 const _ = require("lodash");
 const https = require("https");
 const UUID = require("uuid");
 
+import generateHtml from "./generateHtml";
+import Article from "../browser/components/article";
+import ErrorPage from "../browser/components/error";
+
+require("dotenv").config();
+
 app.use("/dist", express.static(`${__dirname}/../client`));
 
 app.get("/article/:id", async (req, res) => {
   const articleId = req.params.id;
   if (articleId && UUID.validate(articleId)) {
     const authHeaders = () => {
       if (req.headers.authuser && UUID.validate(req.headers.authuser) && req.headers.authorization) {
```

Comment:
@vapaj `authUser` rather than `authuser` maybe, to make the naming consistent with `authHeaders` on the line before?

Reply:
The `req` object comes from [express](https://expressjs.com/), so we don't have control over that unfortunately.